.github/CODEOWNERS

```diff
@@ -1,3 +1,9 @@
-*                            @rot1024
+* @rot1024
```

Comment:
I don't think @rot1024 will want to be codeowner of the whole repo. Should ask him what directories he wants to be

Reply:
You may make me the overall code owner for now. Most changes will be made to web and server, which already have their own code owners assigned. There will not be many opportunities for me to review.